CliClient/locales/ko.po

```diff
@@ -3201,105 +3208,3 @@ msgstr "OneDrive 로그인"
 #: /Users/tessus/data/work/joplin/Tools/../ReactNativeClient/lib/components/screens/search.js:163
 msgid "Search"
 msgstr "검색"
-
-#~ msgid "Global zoom percentage"
-#~ msgstr "전체 확대 비율"
-
-#~ msgid "Exporting to \"%s\" as \"%s\" format. Please wait..."
-#~ msgstr "\"%s\"에 \"%s\" 포맷으로 내보내는 중입니다. 잠시만 기다려주세요..."
-
-#~ msgid "Could not export notes: %s"
-#~ msgstr "내보낼 노트를 찾을수 없습니다: %s"
-
-#~ msgid "All potential ports are in use - please report the issue at %s"
-#~ msgstr ""
-#~ "모든 가능한 포트들이 사용중에 있습니다 - %s 에 대한 문제를 알려주세요"
-
-#~ msgid "Synchronisation is already in progress. State: %s"
-#~ msgstr "동기화가 이미 진행되고 있습니다. 상태: %s"
-
-#~ msgid "Confirm master password:"
-#~ msgstr "마스터 키를 넣어주세요:"
-
-#~ msgid "Confirm password"
-#~ msgstr "마스터 키를 넣어주세요"
-
-#, fuzzy
-#~ msgid "Missing required argument: note"
-#~ msgstr "필요한 인자가 없습니다: %s"
-
-#~ msgid "Starting to edit note. Close the editor to get back to the prompt."
-#~ msgstr "노트 편집을 시작합니다. 프롬프트로 가시려면 편집기를 종료해주세요."
-
-#~ msgid "Note has been saved."
-#~ msgstr "노트가 저장되었습니다."
-
-#~ msgid "Synchronisation status"
-#~ msgstr "동기화 상태"
-
-#~ msgid "General Options"
-#~ msgstr "일반 옵션"
-
-#~ msgid "Encryption options"
-#~ msgstr "암호화 옵션"
-
-#~ msgid "Encryption Options"
-#~ msgstr "암호화 옵션"
-
-#~ msgid "Clipper Options"
-#~ msgstr "수집기 옵션"
-
-#~ msgid ""
-#~ "Unknown item type downloaded - please upgrade Joplin to the latest version"
-#~ msgstr ""
-#~ "알려지지 않은 타입을 다운로드 했습니다 - Joplin을 최신 버전으로 업데이트 "
-#~ "해주세요"
-
-#, fuzzy
-#~ msgid "Permission to write to external storage"
-#~ msgstr "카메라 사용 허가"
-
-#~ msgid "Cancel synchronisation"
-#~ msgstr "동기화 취소"
-
-#~ msgid "Hide metadata"
-#~ msgstr "메타데이터 숨김"
-
-#~ msgid "Show metadata"
-#~ msgstr "메타데이터 표시"
-
-#~ msgid "Delete notebook"
-#~ msgstr "노트북 삭제"
-
-#~ msgid ""
-#~ "Click on the (+) button to create a new note or notebook. Click on the "
-#~ "side menu to access your existing notebooks."
-#~ msgstr ""
-#~ "(+) 버튼을 눌러 새 노트 및 노트북을 만드세요. 측면 메뉴를 눌러 기존의 노트"
-#~ "북을 관리하세요."
-
-#~ msgid ""
-#~ "You currently have no notebook. Create one by clicking on (+) button."
-#~ msgstr "노트북이 없습니다. (+) 버튼을 눌러 새로 만드세요."
-
-#~ msgid "Welcome"
-#~ msgstr "환영합니다"
-
-#~ msgid "Separate each tag by a comma."
-#~ msgstr "각각의 태그를 쉼표로 구분합니다."
-
-#~ msgid "Some items cannot be decrypted."
-#~ msgstr "일부 항목들은 암호화할 수 없습니다."
-
-#~ msgid ""
-#~ "The path to synchronise with when file system synchronisation is enabled. "
-#~ "See `sync.target`."
-#~ msgstr ""
-#~ "파일 시스템 동기화가 활성화된 경우에 동기화를 할 경로입니다. `sync.target`"
-#~ "를 참조하세요."
-
-#~ msgid "State: %s."
-#~ msgstr "상태: %s."
-
-#~ msgid "A notebook with this title already exists: \"%s\""
-#~ msgstr "같은 제목의 노트북이 이미 있습니다: \"%s\""
```

Comment:
Do not remove these old translations.

Reply:
Yep! I have pushed new commit to restore that!